api/health/service.go

```diff
@@ -33,12 +38,24 @@ func (s *Service) Readiness(_ *http.Request, _ *struct{}, reply *APIReply) error
 }
 
 // Health returns a summation of the health of the node
-func (s *Service) Health(_ *http.Request, _ *struct{}, reply *APIReply) error {
+func (s *Service) Health(_ *http.Request, args *HealthArgs, reply *APIReply) error {
 	s.log.Debug("API called",
```

Comment:
Can we include the `subnetIDs` in the log here? Should be able to use a `zap.Stringers` field.

Reply:
added tags